### DISCUSSION TITLE
About the Kronecker symbol

### DISCUSSION THREAD

**Discussion by Nuwanted** [idaholab/moose#26333] [Q&A Modules: Phase field]
Hello all， I have recently been researching the problem of strain.  In the free energy function, I need to calculate the value of the volumetric strain, but I am not sure how to implement the Kronecker symbol in MOOSE.  Thank you very much for your help.

**Comment by GiudGiud** [comment]
Hello

You could use an "if" condition in a parsed expression.

However unless it gets evaluated at just the right point it will never be true. 

It s a good question for @dschwen
